style.css

```diff
@@ -419,14 +419,177 @@ input:focus {
   border-color: rgb(98, 108, 201);
 }
 
+.headerlish {
+  display: none;
+}
+
 @media (min-width: 375px) {
   .content-section {
     background-size: cover;
   }
 }
 
+/* desktop version */
+
 @media (min-width: 768px) {
+  .icon-Union {
+    display: none;
+  }
+
+  .topBar-nav {
+    position: sticky;
+    top: 0;
+    background-color: white;
+  }
+
+  .headerlist {
+    display: flex;
+    gap: 8px;
+    list-style-type: none;
+    padding-right: 20px;
+    font-family: var(--primary-font);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    position: sticky;
+    top: 0;
+  }
+
+  .headerlist a:hover {
+    color: rgb(24, 155, 160);
+  }
+
   .content-section {
+    background-image: url(img/Header\ bg-desktop.png);
+    background-size: contain;
+    display: flex;
+    height: 718px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
- [ ] Great job on styling the desktop version of your project :+1:. However, it is essential to match the design as closely as possible to the Figma template. Here are some recommended actions to help achieve this goal:
- Check the padding and margins of your navbar. It appears to be too close to the edges. Adjusting these values will help create a more balanced and visually appealing design.
- Verify the color of your links "Portfolio," "About," and "Contact." The color should be #344563, as per the template. Matching this color will help maintain consistency and ensure a cohesive design.
- Check the color of the "Let's Connect" button. Make sure it matches the color of the button in the Figma template to maintain consistency.
- Ensure that the background image takes up the entire page as shown in the template. This will help create a more immersive experience for users.
- The bottom left corner of the page should be rounded. Adding this rounded corner will help align the design with the template.

By following these recommendations, you can achieve a design that closely matches the Figma template and create a polished, professional-looking project.

**Your design**

![Screenshot from 2023-04-01 15-55-19](https://user-images.githubusercontent.com/105117832/229313755-9d6df568-ebfa-44c4-9abf-189217d522e6.png)

**Figma**

![Screenshot from 2023-04-01 15-59-46](https://user-images.githubusercontent.com/105117832/229313767-f68ae3e7-e6d6-494b-8b78-b7a2332430cd.png)